src/detector.jl

```diff
@@ -190,8 +190,8 @@ function construct_ssd_simulation(det_meta::PropDict, env::Environment, simulato
         error("Only 2D is supported up to now.")
     end
     sim.world = begin
-        crystal_radius = to_SSD_units(T, det_meta.geometry.radius_in_mm, u"mm")
-        crystal_height = to_SSD_units(T, det_meta.geometry.height_in_mm, u"mm")
+        crystal_radius = SolidStateDetectors._parse_value(T, det_meta.geometry.radius_in_mm, u"mm")
```

Comment:
We shouldn't use internal functions of SolidStateDetectors.

If we need that, we should make it part of the public SolidStateDetectors API.

Reply:
Ok, I'm removing this from this PR then.